packages/mds-db/index.ts

```diff
@@ -1178,7 +1170,7 @@ async function readTrips(
   }
 
   const { rows: trips } = await exec(
-    `SELECT * FROM ${schema.TRIPS_TABLE} ${where} ORDER BY recorded ASC, sequence ASC${
+    `SELECT * FROM ${schema.TRIPS_TABLE} ${where} ORDER BY recorded ASC, provider_trip_id ASC${
```

Comment:
`provider_trip_id` seems like a kind of odd thing to be sorting by... Necessary?

Reply:
Sorts are `recorded IDX + PK`